Formula/s/seaweedfs.rb

```diff
@@ -30,6 +30,15 @@ def install
       -X github.com/seaweedfs/seaweedfs/weed/util.COMMIT=#{Utils.git_head}
     ]
     system "go", "build", *std_go_args(ldflags:, output: bin/"weed"), "./weed"
+    (var/"seaweedfs").mkpath # Ensure the seaweedfs directory exists
```

Comment:
this can be moved to postinstall

Reply:
Sure no problem! I'll fix when I get a moment at the keys.

Reply:
Ok, I moved it to a new post_install block. Let me know if I got that right/wrong. Thank you for the review!